charts/prometheus-cloudwatch-exporter/templates/deployment.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     chart: {{ template "prometheus-cloudwatch-exporter.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- if .Values.deployment.labels }}
+  {{ toYaml .Values.deployment.labels | indent 4 }}
+{{- end }}
```

Comment:
I you move these down to `template.metadata.labels` and name them `pod.labels` then it's fine for me.